api/redisfailover/v1/defaults.go

```diff
@@ -3,6 +3,7 @@ package v1
 const (
 	defaultRedisNumber    = 3
 	defaultSentinelNumber = 3
+	defaultSentinelExporterImage = "leominov/redis_sentinel_exporter:latest"
```

Comment:
Could you set an immutable tag as default value? so we can avoid unexpected breaking changes.

Reply:
OK, change default sentinel exporter image to  leominov/redis_sentinel_exporter:1.3.0.